gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -439,6 +441,18 @@ static GtkActionEntry gnc_plugin_page_register_actions [] =
         N_("Open a register report for the selected Transaction"),
         G_CALLBACK (gnc_plugin_page_register_cmd_transaction_report)
     },
+
+    /* Windows menu */
+    {
+        "WindowsSaveLayoutAction", NULL, "_Save Layout for this Register type", NULL,
+        "Save this layout for this register type to the default layout",
+        G_CALLBACK (gnc_plugin_page_register_cmd_save_layout)
+    },
+    {
+        "WindowsResetLayoutAction", NULL, "_Reset Layout for this Register type", NULL,
+        "Reset column widths and remove the default layout for this register type",
```

Comment:
These strings should be marked for translation.
And like with the business options, I propose to use
"Use as Default Layout for this Register Type"
"Reset Default Layout for this Register Type"
Or more concisely
"Use as Default Layout"
"Reset Default Layout"

And tooltips:
"Use the current layout as default for all registers of this type"
"Reset default layout for all registers of this type back to built-in default and update page accordingly"

Is there a way to insert the register type in here ?

Reply:
I think that was a copy and paste error from the invoice stuff, I wondered about register type, will have a think after making other changes.